cylc.tmLanguage.json

```diff
@@ -1,6 +1,12 @@
 {
     "scopeName": "source.cylc",
     "$schema": "https://raw.githubusercontent.com/martinring/tmlanguage/master/tmlanguage.json",
+    "fileTypes": [
+        "suite.rc",
+        "suite.rc.processed",
+        "cylc",
+        "flow.rc"
```

Comment:
Might as well add `.cylc` now.

Reply:
If specifying just the extension, there's no need for the dot

Reply:
Ah, ok